Source/Fuse.Common/UpdateManager.uno

```diff
@@ -82,8 +81,18 @@ namespace Fuse
 		public List<UpdateListener> Onces = new List<UpdateListener>();
 		public List<UpdateListener> OncesPending = new List<UpdateListener>();
 		
-		public List<UpdateAction> PhaseDeferredActions = new List<UpdateAction>();
-		public int PhaseDeferredActionsAt = -1;
+		public Dictionary<int, Queue<UpdateAction>> PhaseDeferredActions = new Dictionary<int, Queue<UpdateAction>>();
```

Comment:
Yes, the restriction on the API is okay. It's structured such that it expects precse levels/groupings anyway. Passing random integers would basically result in undefined behaviour in some situations.